tests/integration/targets/aci_l2out_extepg_to_contract/tasks/main.yml

```diff
@@ -0,0 +1,198 @@
+  assert:
+    that:
+    - l2extepg_again is not changed
+    - l2extepg.current.0.l2extInstP.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out/instP-ansible_extepg"
+
+- name: Add another L2 external end point group
+  aci_l2out_extepg:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l2out: ansible_l2out_2
+    extepg: ansible_extepg_2
+    description: Ansible external epg
+    qos_class: level1
+    preferred_group: True
+    state: present
+  register: l2extepg_2
+
+- name: Verify l2extepg_2
+  assert:
+    that:
+    - l2extepg_2.current.0.l2extInstP.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out_2/instP-ansible_extepg_2"
+
+- name: Bind External End Point Groups to Contracts
+  aci_l2out_extepg_to_contract:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l2out: ansible_l2out
+    extepg: ansible_extepg
+    contract: ansible_contract
+    contract_type: provider
+    state: present
+  register: bind_extepg_provider_contract
+
+- name: Verify bind_extepg_provider_contract
+  assert:
+    that:
+    - bind_extepg_provider_contract.current.0.fvRsProv.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out/instP-ansible_extepg/rsprov-ansible_contract"
+
+- name: Bind second External End Point Groups to Contracts
+  aci_l2out_extepg_to_contract:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l2out: ansible_l2out_2
+    extepg: ansible_extepg_2
+    contract: ansible_contract2
+    contract_type: provider
+    state: present
+  register: bind_extepg_provider_contract
+
+- name: Verify bind_extepg_provider_contract
+  assert:
+    that:
+    - bind_extepg_provider_contract.current.0.fvRsProv.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out_2/instP-ansible_extepg_2/rsprov-ansible_contract2"
+
+- name: Query the External End Point Groups
+  aci_l2out_extepg_to_contract:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l2out: ansible_l2out
+    extepg: ansible_extepg
+    contract: ansible_contract
+    contract_type: provider
+    state: query
+  register: query_extepg
+
+- name: Verify query_extepg
+  assert:
+    that:
+    - query_extepg is not changed
+    - query_extepg.current.0.fvRsProv.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out/instP-ansible_extepg/rsprov-ansible_contract"
+
+- name: Query all the External End Point Groups
+  aci_l2out_extepg_to_contract:
+    <<: *aci_info
+    contract_type: provider
+    state: query
+  register: query_all
+
+- name: Remove existing contract to External End Point Groups
+  aci_l2out_extepg_to_contract:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l2out: ansible_l2out
+    extepg: ansible_extepg
+    contract: ansible_contract
+    contract_type: provider
+    state: absent
+  register: remove_contract_extepg
+
+- name: Verify remove_contract_extepg
+  assert:
+    that:
+    - remove_contract_extepg.previous.0.fvRsProv.attributes.dn == "uni/tn-ansible_tenant/l2out-ansible_l2out/instP-ansible_extepg/rsprov-ansible_contract"
```

Comment:
verify remove_contract_extepg is changed
verify remove_contract_extepg.current == []